### ISSUE TITLE
Vim-ipython-cell vs jupyter-vim

### ISSUE THREAD

**Issue by quantum-booty** [hanschen/vim-ipython-cell#13]
How does this plugin differ from the [jupyter-vim ](https://github.com/jupyter-vim/jupyter-vim) plugin and why should I use one over the other?

**Comment by hanschen** [comment]
`jupyter-vim` seems to be based on `vim-ipython` and probably uses similar mechanisms to send the code to jupyter. One caveat seems to be that the output won't show up in the console version until you press Enter (see README). Furthermore, it seems to show all the code that's sent over, while `vim-ipython-cell` shows only `%paste -q` no matter how much code your cell contains. Finally, it's probably easier to use virtual environments with this plugin, since it doesn't require that Vim and IPython use the same Python interpreter, unlike the `jupyter-vim` plugin. The downside of this plugin is that it doesn't support the qtconsole client and that you have to run vim and IPython in e.g. tmux or screen.

In short, if you want to use the qtconsole client, choose `jupyter-vim` (or similar plugins). Otherwise try both and see which one you prefer. This plugin is probably easier to set up if you're already familiar with e.g. tmux.
